packages/react-scripts/scripts/init.js

```diff
@@ -216,20 +216,24 @@ module.exports = function(
   // Install react and react-dom for backward compatibility with old CRA cli
   // which doesn't install react and react-dom along with react-scripts
   if (!isReactInstalled(appPackage)) {
-    args = args.concat(['react', 'react-dom']);
+    args = args.concat(['react@16.4.2', 'react-dom@16.4.2']);
   }
 
   // Install template dependencies, and react and react-dom if missing.
-  if ((!isReactInstalled(appPackage) || templateName) && args.length > 1) {
-    console.log();
-    console.log(`Installing template dependencies using ${command}...`);
+  // if ((!isReactInstalled(appPackage) || templateName) && args.length > 1) {
+  //  console.log();
+  //  console.log(`Installing template dependencies using ${command}...`);
+  console.log(
+    `Installing ${chalk.cyan('Backpack')} dependencies using ${command}`
+  );
+  console.log();
 
-    const proc = spawn.sync(command, args, { stdio: 'inherit' });
-    if (proc.status !== 0) {
-      console.error(`\`${command} ${args.join(' ')}\` failed`);
-      return;
-    }
+  let proc = spawn.sync(command, args, { stdio: 'inherit' });
+  if (proc.status !== 0) {
+    console.error(`\`${command} ${args.join(' ')}\` failed`);
+    return;
```

Comment:
Superfluous whitespace change

Reply:
Not sure what you mean here? Can you be elaborate?